weepickle-implicits/src/main/scala/com/rallyhealth/weepickle/v1/implicits/Froms.scala

```diff
@@ -15,4 +15,5 @@ trait Froms extends DefaultFroms { this: Annotator =>
     override def transform0[V](v: Instant, out: Visitor[_, V]): V = out.visitTimestamp(v)
   }
   implicit val FromDate: From[Date] = FromInstant.comap(_.toInstant)
+  implicit val FromZoneId: From[ZoneId] = FromString.comap[ZoneId](_.toString)
```

Comment:
I don't think adding members to traits will pass MiMa, but they can go into `LowPriorityImplicits` since it's a class. The end result should be the same.

Reply:
Moved the new implicits into `LowPriorityImplicits`